app/assets/stylesheets/components/_avatar.scss

```diff
@@ -0,0 +1,27 @@
+.avatars-list {
+  @include row;
+
+  li {
+    margin-bottom: $small-spacing;
+
+    @include media(max-width 24.9375rem) {
```

Comment:
It's okay to use 399px here if that's what we're really aiming at. Obvious is better.